docs/query_tool.md

```diff
@@ -45,13 +45,15 @@ This TSV contains the following columns:
 Note that this ID is not guaranteed to be persistent across versions of a graph/across graphs, but will always be identical across a pair of query tool result files. 
 _This column can be used as the key to join the dataset-level and participant-level results TSVs for a given query result, if needed._
 - `SubjectID`: subject label
+- `SessionID`: the label of the session (can be empty if there is no session)
```

Comment:
```suggestion
- `SessionID`: the label of the session
```

I don't think `SessionID` can actually be empty, since all phenotypic/imaging data for a subject will be created under either a phenotypic or imaging session (if a session doesn't exist in the raw data, we just create one in the CLI).